primitives/bridge/src/container_token_to_ethereum_message_exporter.rs

```diff
@@ -0,0 +1,1679 @@
+// Copyright (C) Moondance Labs Ltd.
+// This file is part of Tanssi.
+
+// Tanssi is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Tanssi is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Tanssi.  If not, see <http://www.gnu.org/licenses/>
+
+use {
+    alloc::vec::Vec,
+    core::{iter::Peekable, marker::PhantomData, slice::Iter},
+    frame_support::{ensure, traits::Get},
+    parity_scale_codec::{Decode, Encode},
+    snowbridge_core::{AgentId, ChannelId, TokenId},
+    snowbridge_outbound_queue_primitives::v1::message::{Command, Message, SendMessage},
+    sp_core::H160,
+    sp_runtime::traits::MaybeEquivalence,
+    xcm::latest::SendError::{MissingArgument, NotApplicable, Unroutable},
+    xcm::prelude::*,
+    xcm_executor::traits::ExportXcm,
+};
+
+pub struct EthereumBlobExporter<
```

Comment:
I would change the name of this directly to ContainerEthereumBlobExporter

Reply:
Updated here: https://github.com/moondance-labs/tanssi/pull/1100/commits/fd2bed662b7a6e5eacdb2f9572635fca8e08e337